cfdm/data/aggregatedarray.py

```diff
@@ -2,14 +2,15 @@
 from itertools import accumulate, product
 
 import numpy as np
-from uritools import isuri, uricompose
 
 from ..functions import dirname
 from . import abstract
 from .fragment import FragmentFileArray, FragmentUniqueValueArray
 from .netcdfindexer import netcdf_indexer
 from .utils import chunk_locations, chunk_positions
 
+# from uritools import isuri, uricompose
+
```

Comment:
Left over from dev?

```suggestion
```

(plus extra line also introduced - suggestion won't let me include that still 😬 )